packages/@aws-cdk/aws-lambda/test/test.vpc-lambda.ts

```diff
@@ -43,6 +43,81 @@ export = {
 
       test.done();
     }
+	
+	  public 'has securitygroup that is passed in props'(test: Test) {
```

Comment:
Indentation is still off across this file.

Reply:
Finally sorted...